app/controllers/articles_controller.rb

```diff
@@ -4,7 +4,9 @@ class ArticlesController < ApplicationController
   before_action :set_article, only: [:show, :edit, :update, :destroy]
 
   def index
-    @articles = Article.all.order('date DESC')
+    @articles = Article.all.
+      paginate(:page => params[:page], :per_page => 10).
```

Comment:
Can we use the Ruby 1.9 hash syntax here? ie `page: params[:page], per_page: 10`